### ISSUE TITLE
[bug]: textual inversion produce black images

### ISSUE THREAD

**Issue by Brandejs** [invoke-ai/InvokeAI#3279]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

macOS

### GPU

CPU (M2 Pro 12-core CPU, 19-core GPU, 16-core Neural Engine)

### What version did you experience this issue on?

2.3.4.post1

### What happened?

I ran textual inversion training. and had my face trained with this setup. Training data is already cropped and resized to 512x512px around 13 images.
![image](https://user-images.githubusercontent.com/5411927/234784879-d092e052-38f7-4574-af34-239f4b8436d7.png)

Everything went well. After completion if I use in web UI and add textual-inevrsion token. Only black images will be produced. Even if I give the token away, it still produces black images. The only thing that helps is a complete restart of the application. The application then works until I use the token.

![image](https://user-images.githubusercontent.com/5411927/234785425-589514db-6934-4f1d-8330-cb9ad8f77b07.png)

### Additional context

I am a beginner who is still trying to understand the concept and how it works.

**Comment by rpfile** [comment]
have seen the same thing. haven't updated InvokeAI for a very long time until a few days ago when i tried 2.3.4.post1. one thing i do notice is that the loss pretty quickly becomes NaN while training. this might have something to do with the problem.

**Comment by Brandejs** [comment]
Yes, I can confirm that. I started a new training and the loss changed instantly to NaN  

![image](https://user-images.githubusercontent.com/5411927/235074283-9d107dfb-85d5-4054-9391-d8f6864d3e31.png)
